Assets/Scripts/BulletManagement/SlipTimeEmitter.cs

```diff
@@ -6,7 +6,7 @@ namespace BulletManagement
     /// <inheritdoc cref="BulletManagement.EmitterBase" />
     public class SlipTimeEmitter : EmitterBase, ISlipTimeAdherent
     {
-        public SlipTimeManager SlipTimeManager => slipTimeManager;
```

Comment:
Because we instantiate emitters from prefabs and we can't save an instance of slip time manager as part of the prefab, we need to be able to set `slipTimeManager` from outside the emitter.